android/README.md

```diff
@@ -182,8 +182,13 @@ find_library(PYTORCH_LIBRARY pytorch_jni
   PATHS ${PYTORCH_LINK_DIRS}
   NO_CMAKE_FIND_ROOT_PATH)
 
+find_library(FBJNI_LIBRARY fbjni
```

Comment:
Not in this PR: have you thought about providing a cmake file for client project to import useful library variables without manually creating/maintaining them?

I've seen this pattern for some OSS library, including PyTorch itself. Here is the public cmake file we generate: https://github.com/pytorch/pytorch/blob/master/cmake/TorchConfig.cmake.in

And in client project's cmake it can use `find_package` to locate the library's cmake and obtain `$TORCH_INCLUDE_DIRS` and `$TORCH_LIBRARIES` variables directly: https://github.com/pytorch/pytorch/blob/master/test/mobile/custom_build/CMakeLists.txt#L8

In this specific example, we don't need to maintain the dozen of static libraries as we do elsewhere: https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/CMakeLists.txt#L96-L105

Just an idea - not sure if it's appropriate/common for native android projects.

Reply:
Thanks. This is a good idea, I will try to create cmake config files, to use just find_package on client side